texastribune/lunchbox
# Customize it

#### What's this PR do?

Adds our colors, logos, and other customizations to Waterbug, Factlist, and Quotable.
#### Why are we doing this? How does it help us?

Improves our social media branding.
#### How should this be manually tested?

Follow the steps to set up the project and get it running in the repo.

Once you have it running, test out Waterbug. Try uploading an image file, or using an image url. Select a copyright holder and enter example info. Check out the different crops, and drag your sample image around on the Twitter and Instagram crops to see how it works. Check that the Twitter image is 1024x512, as requested by editorial. Check that the Instagram image is 612x612, as requested by editorial. Toggle between different logos and logo colors. Then save the image and open to see how it looks.

Then check out Factlist. Toggle the themes, change the font size, play with different text, play with the aspect ratio, and toggle the timestamp. Then save the image and open to see how it looks.

Then check out Quotable, which is similar to Pixelcite. It has the same themes as Factlist. Change the aspect ratio, and toggle quotation marks. Save the image and open to see how it looks.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

n/a
#### What are the relevant tickets?

Sprint task: https://basecamp.com/1866004/projects/1175549/todos/193422135
#### Next steps?

Rebuild the desktop app with Electron and share with editorial/anyone else who wants to use it.
#### Smells?
#### TODOs:
- [x] Share with editorial to get feedback on any changes
- [x] Get art approval
- [x] Customize Quotable
- [x] Update fonts
#### Has the relevant documentation/wiki been updated?

TODO add to the wiki
#### Technical debt note
